manifests/rules/out/puppet.pp

```diff
@@ -1,19 +1,19 @@
 # manage outgoing puppet
 class nftables::rules::out::puppet (
```

Comment:
There's no tests for this class, maybe be add some trivial ones.

Reply:
yeah wasn't sure, whether this should happen in this change

Reply:
added